aws-iot-accountauditconfiguration/src/test/java/com/amazonaws/iot/accountauditconfiguration/TranslatorTest.java

```diff
@@ -68,7 +79,36 @@ void translateNotificationsFromIotToCfn_NonEmpty() {
     }
 
     @Test
-    void translateNotificationsFromIotToCfn_NullIn_EmptyOut() {
-        assertThat(Translator.translateNotificationsFromIotToCfn(null)).isEmpty();
+    void translateNotificationsFromIotToCfn_NullIn_NullOut() {
+        assertThat(Translator.translateNotificationsFromIotToCfn(null)).isNull();
+    }
+
+    @Test
+    void translateNotificationsFromIotToCfn_NonExistentKeyFromIot_VerifyIgnoredInTranslation() {
+        assertThat(Translator.translateNotificationsFromIotToCfn(AUDIT_NOTIFICATION_TARGET_IOT_NON_EXISTENT_KEY))
+                .isEqualTo(AUDIT_NOTIFICATION_TARGET_CFN);
+    }
+
+    @Test
+    void translateChecksFromCfnToIot_NullIn_EmptyOut() {
+        ResourceModel input = ResourceModel.builder().build();
+        assertThat(Translator.translateChecksFromCfnToIot(input)).isEmpty();
     }
+
+    @Test
+    void translateChecksFromCfnToIot_AllChecksArePresent_VerifyTranslation() {
+        Map<String, software.amazon.awssdk.services.iot.model.AuditCheckConfiguration> expectedResult =
+                DESCRIBE_RESPONSE_ALL_CHECKS_PRESENT.auditCheckConfigurations();
+        ResourceModel input = ResourceModel.builder()
+                .auditCheckConfigurations(DESCRIBE_RESPONSE_ALL_CHECKS_PRESENT_CFN).build();
+        assertThat(Translator.translateChecksFromCfnToIot(input)).isEqualTo(expectedResult);
+    }
```

Comment:
We need another test that covers each check in isolation, what if in the test you are enabling LOGGING check for the case where CA_CERTIFICATE_EXPIRING_CHECK check was intended to be enabled?
* For each check let's make sure that the case where it is enabled and all other checks are disabled works.

Reply:
Added.